recipes/sol2/config.yml

```diff
@@ -1,4 +1,5 @@
 versions:
-  3.0.3:
-    folder: 3.x.x
-    
```

Comment:
please tell ,e what the gab between line 4 and 5 on the right hand side means, is it the reason for all the problems ?